benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaConsumerBenchmarks.scala

```diff
@@ -69,7 +69,7 @@ object ReactiveKafkaConsumerBenchmarks extends LazyLogging {
         batch.updated(elem)
 
       }
-      .mapAsync(1) { m =>
+      .mapAsync(3) { m =>
```

Comment:
@patriknw right, thanks
@13h3r I don't think it's possible to commit in order 2,1 because `commitAsync` calls will always be performed in correct order.